scripts/zones/Mamook/mobs/Gulool_Ja_Ja.lua

```diff
@@ -6,61 +6,54 @@
 require("scripts/globals/titles");
 require("scripts/globals/status");
 require("scripts/zones/Mamook/MobIDs");
+mixins = {require("scripts/mixins/job_special")};
 
 function onMobSpawn(mob)
-    mob:setLocalVar("1Hour", math.random(3,5)); 
     mob:setMod(MOD_DOUBLE_ATTACK, 20);
+    mob:setMobMod(MOBMOD_DRAW_IN, 2);
 end;
 
 function onMobEngaged(mob,target)
-    for i = 17043876, 17043879, 1 do
+    for i = GULOOL_JA_JA + 1, GULOOL_JA_JA + 4 do
         SpawnMob(i):updateEnmity(target);
     end
 end;
 
 function onMobFight(mob, target)
-    local petIDs = {17043876,17043877,17043878,17043879};
-    local petStatus = {GetMobAction(petIDs[1]),GetMobAction(petIDs[2]),GetMobAction(petIDs[3]),GetMobAction(petIDs[4])};
-    local HPP = mob:getHPP();
-
-    if (mob:getLocalVar("use1Hour") == 0) then
-        if (HPP <= mob:getLocalVar("1Hour")) then
-            mob:useMobAbility(731); -- Mijin Gakure
-            mob:setLocalVar("use1Hour", 1);
-        end
-    end
+    local petIDs = {GULOOL_JA_JA + 1, GULOOL_JA_JA + 2, GULOOL_JA_JA + 3, GULOOL_JA_JA + 4};
+    local petStatus = {GetCurrentAction(petIDs[1]),GetCurrentAction(petIDs[2]),GetCurrentAction(petIDs[3]),GetCurrentAction(petIDs[4])};
```

Comment:
almost right.  getCurrentAction is a lua_baseentity binding.  entire onMobFight could be simply:
https://pastebin.com/raw/dtmqvQm7